contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -493,6 +589,8 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         }
         _saveOperatorStuckPenaltyStats(_nodeOperatorId, stuckPenaltyStats);
         emit StuckValidatorsCountChanged(_nodeOperatorId, _stuckValidatorsCount);
+
+        _updateTotalMaxValidatorsCount(_nodeOperatorId);
     }
 
     function _updateRefundValidatorsKeysCount(uint256 _nodeOperatorId, uint64 _refundedValidatorsCount) internal {
```

Comment:
See comments for `_updateStuckValidatorsCount()`

Reply:
1) `_requireValidRange` is correct, no more than 'DEPOSITED' kays can be marked as 'EXITED'
2) event added